cf/i18n/resources/es-es.all.json

```diff
@@ -3257,7 +3257,7 @@
   },
   {
     "id": "Perform a simple check to determine whether a route currently exists or not",
-    "translation": "Realice una comprobación simple para determinar si existe o no en este momento una ruta"
+    "translation": "Realice una comprobación simple para determinar si existe o no en este momento una ruta."
```

Comment:
This is the description of the `check-route` command. Please don't add a period at the end of single-sentence command descriptions.